jujugui/static/gui/src/app/app.js

```diff
@@ -491,7 +491,7 @@ YUI.add('juju-gui', function(Y) {
       let password = null;
       let macaroons = null;
       let credentials = this.user.controller;
-      if (credentials.areAvailable) {
+      if (credentials && credentials.areAvailable) {
```

Comment:
As controller is a property of user, this check should never be required, so I am a bit confused.
See https://github.com/juju/juju-gui/blob/develop/jujugui/static/gui/src/app/user/user.js#L44
The areAvailable attribute should always be defined, even if we don;t have credentials.